src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -592,6 +592,14 @@ private function addService($id, $definition)
             $return[] = sprintf("@throws InactiveScopeException when the '%s' service is requested while the '%s' scope is not active", $id, $scope);
         }
 
+        if ($definition->isDeprecated()) {
+            if ($return && 0 === strpos($return[count($return) - 1], '@return')) {
+                $return[] = '';
+            }
+
+            $return[] = '@deprecated';
```

Comment:
IMO, the generic message should never be used, because it's, well, generic. In Symfony at least, I'd advocate never using the default message but always provide a custom one.
Adding the message after `@deprecated` would be required for e.g. the Symfony plugin for phpstorm to easily hint devs about the deprecation.
I'd be in favor of adding back the message here, and forbidding messages that match `(\r|\n*/)`. An InvalidArgumentException should be thrown by setDeprecated then.